src/core/services/sites.ts

```diff
@@ -2195,6 +2209,7 @@ export type CoreSiteBasicInfo = {
     badge?: number; // Badge to display in the site.
     siteHomeId?: number; // Site home ID.
     loggedOut: boolean; // If Site is logged out.
+    isDemoModeSite: boolean;
```

Comment:
```suggestion
    demoMode: boolean;
```

I think it's better to use this nomenclature to distinguish between booleans and methods. Whilst reviewing the code, I saw this in a couple of templates and initially I though it was a typo missing the parentheses. Angular won't complain if a method is used without a call, given that it's always truthy. And we are using this same name for methods elsewhere in the codebase, so I think it's better to keep them distinct to avoid confusion.

Reply:
This morning I started working on the next issue, MOBILE-4460, and this property will be removed in that issue xD Now there are more conditions to determine if a URL should be displayed, it's not just if the site is demo, so I decided to remove this property and always use a method instead.

So if you don't mind we can leave this name as it is since it will be removed soon.